tests/test_freqtradebot.py

```diff
@@ -3545,8 +3545,33 @@ def test_get_real_amount_no_trade(default_conf, buy_order_fee, caplog, mocker, f
                    caplog)
 
 
-def test_get_real_amount_stake(default_conf, trades_for_order, buy_order_fee, fee, mocker):
-    trades_for_order[0]['fee']['currency'] = 'ETH'
+@pytest.mark.parametrize(
+    'fee_par,fee_reduction_amount,use_ticker_rate,expected_log', [
+        # basic, amount does not change
+        ({'cost': 0.008, 'currency': 'ETH'}, 0, False, None),
```

Comment:
While i updated these tests to use a "fee" dict now ... i've discovered that this testcase does not test the same thing than the existing test.

`test_get_real_amount_stake` (which updates currency to 'ETH') keeps the cost of 0.008 from the original trades_for_order - so a correct test (correctly replicating previous behaviour) would be `{'cost': 0.008, 'currency': 'ETH'}`.

I've now adjusted this test accordingly (check the last commit for necessary changes) ... and based on my comparisons, the remaining tests should be ok, so for me, this would be OK now.

Reply:
ok, can you merge this if there's no other problems?

Reply:
i can - but wanted to give both you and the original author a chance to look at it.

Also - please stop pushing for merges or reviews. 
I know it takes patience to wait for reviews or merges, but to be honest, freqtrade is quite good in that regards (in other sometimes very active repositories (multiple merges per day) i've contributed to, you can wait for months for even an initial review).
It's putting unnecessary pressure on other people - which might be fine in a company where it's people's main work to do this - but it's not ok for open source projects, where it's people's free time that is invested.
Thanks for your understanding.

Reply:
if it's fine by you it's fine by me

Reply:
Yes - but when editing some parts of user PR's - it's either small adjustments (Right before merging) - or bigger things (like this).
The 2nd case is where i always like to get feedback / thoughts from the author - maybe there's a reason/thought behind i didn't grasp so far - and am now introducing another problem i didn't see yet.

Reply:
ok, sounds good